From 1208afdfdd907bf139168eecf82bd3aec9229ce3 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:41:30 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(4163d656b)..lineage-16.0-android-9.0.0_r31(7aa3e1b5c) (DO
 NOT SUBMIT)

Bill Yi (2):
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (2):
 - Snap for 5044688 from e2e6ddb6e04bd48a7ee2cda9bac5fa893a2a3f59 to
   pi-qpr2-release
 - Snap for 5115288 from ef4599d15797283592d0f82fcb03ab3d1e59111d to
   pi-qpr2-release

Change-Id: If1f09c9178c80f17522bb3aa4b50e76436ba974a

Change-Id: Id14132cd48e7dd943cc3ded70c450bd3445f7cb3
---
 res/values-bn/strings.xml | 2 +-
 res/values-ca/strings.xml | 2 +-
 res/values-mr/strings.xml | 2 +-
 res/values-pt/strings.xml | 4 ++--
 4 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/res/values-bn/strings.xml b/res/values-bn/strings.xml
index 6708f462e..94e4c83fc 100644
--- a/res/values-bn/strings.xml
+++ b/res/values-bn/strings.xml
@@ -178,7 +178,7 @@
     <string name="alarm_volume_title" msgid="8506245173912428522">"অ্যালার্মের ভলিউম"</string>
     <string name="silent_ringtone_title" msgid="6177287302898634765">"নীরব"</string>
     <string name="unknown_ringtone_title" msgid="6637522029614550112">"অজানা"</string>
-    <string name="alarm_volume_muted" msgid="6658289289695673569">"অ্যালার্মের ভলিউম নিঃশব্দ করা হয়েছে"</string>
+    <string name="alarm_volume_muted" msgid="6658289289695673569">"অ্যালার্মের ভলিউম মিউট করা হয়েছে"</string>
     <string name="unmute_alarm_volume" msgid="2655009276035357497">"সশব্দ করুন"</string>
     <string name="silent_default_alarm_ringtone" msgid="6012855475009670997">"ডিফল্ট অ্যালার্মের রিংটোন হ\'ল নীরব"</string>
     <string name="change_setting_action" msgid="1224356633832430160">"পরিবর্তন করুন"</string>
diff --git a/res/values-ca/strings.xml b/res/values-ca/strings.xml
index 11bc688d1..f72dd94c5 100644
--- a/res/values-ca/strings.xml
+++ b/res/values-ca/strings.xml
@@ -179,7 +179,7 @@
     <string name="silent_ringtone_title" msgid="6177287302898634765">"Silenci"</string>
     <string name="unknown_ringtone_title" msgid="6637522029614550112">"Desconegut"</string>
     <string name="alarm_volume_muted" msgid="6658289289695673569">"S\'ha silenciat el volum de l\'alarma"</string>
-    <string name="unmute_alarm_volume" msgid="2655009276035357497">"Activa el so"</string>
+    <string name="unmute_alarm_volume" msgid="2655009276035357497">"Deixa de silenciar"</string>
     <string name="silent_default_alarm_ringtone" msgid="6012855475009670997">"El so d\'alarma predeterminat està en silenci"</string>
     <string name="change_setting_action" msgid="1224356633832430160">"Canvia"</string>
     <string name="app_notifications_blocked" msgid="7249150789023936889">"Les notificacions del rellotge estan bloquejades"</string>
diff --git a/res/values-mr/strings.xml b/res/values-mr/strings.xml
index 33e9c7512..e3e3df700 100644
--- a/res/values-mr/strings.xml
+++ b/res/values-mr/strings.xml
@@ -244,7 +244,7 @@
     <!-- no translation found for seconds (6550091082396571898) -->
     <skip />
   <string-array name="sw_share_strings">
-    <item msgid="842841032273927988">"आपण खरोखर वेगाचे दानव आहात."</item>
+    <item msgid="842841032273927988">"तुम्ही खरोखर वेगाचे दानव आहात."</item>
     <item msgid="6332879039890727169">"आपल्या कष्टाच्या फळांचा आनंद घ्या."</item>
     <item msgid="815382761274660130">"Androids वेगवान असण्यासाठी ज्ञात आहेत, परंतु आपल्या इतके वेगवान नाहीत!"</item>
     <item msgid="7916250650982813737">"फ्यू."</item>
diff --git a/res/values-pt/strings.xml b/res/values-pt/strings.xml
index 679698512..4cfae1869 100644
--- a/res/values-pt/strings.xml
+++ b/res/values-pt/strings.xml
@@ -61,8 +61,8 @@
     <string name="timer_notifications_hours_seconds" msgid="8168090805113197804">"<xliff:g id="HOURS">%1$s</xliff:g> <xliff:g id="SECONDS"> %4$s</xliff:g><xliff:g id="REMAINING"> %3$s</xliff:g>"</string>
     <string name="timer_notifications_hours_minutes_seconds" msgid="4666176878038730417">"<xliff:g id="HOURS">%1$s</xliff:g><xliff:g id="MINUTES"> %2$s</xliff:g><xliff:g id="SECONDS"> %4$s</xliff:g><xliff:g id="REMAINING"> %3$s</xliff:g>"</string>
     <string name="timer_accessibility_one_minute_added" msgid="6096646178998456004">"Um minuto adicionado ao temporizador, <xliff:g id="TIME">%1$s</xliff:g>"</string>
-    <string name="timer_accessibility_stopped" msgid="2753501043114605474">"Temporizador pausado: <xliff:g id="TIME">%1$s</xliff:g>"</string>
-    <string name="timer_accessibility_started" msgid="8503877052873671219">"Temporizador em execução: <xliff:g id="TIME">%1$s</xliff:g>"</string>
+    <string name="timer_accessibility_stopped" msgid="2753501043114605474">"Timer pausado: <xliff:g id="TIME">%1$s</xliff:g>"</string>
+    <string name="timer_accessibility_started" msgid="8503877052873671219">"Timer em execução: <xliff:g id="TIME">%1$s</xliff:g>"</string>
     <string name="timer_remaining_single" msgid="6782851171906765864">"restante"</string>
     <string name="timer_remaining_multiple" msgid="6052775962092160851">"restantes"</string>
   <string-array name="alarm_set">
-- 
2.17.1

